csharp/test/Microsoft.ML.OnnxRuntime.Tests.NetCoreApp/InferenceTest.netcore.cs

```diff
@@ -1251,6 +1251,87 @@ private void TestModelSerialization()
             }
         }
 
+        private static OrtLoraAdapter CreateLoraAdapterFromFile()
+        {
+            var adapterPath = Path.Combine(Directory.GetCurrentDirectory(), "two_params_lora_model.onnx_adapter");
```

Comment:
Out of interest is the formatting for the ORT LoRA adapter documented somewhere? ideally with an example of how to create/use one ?

Reply:
Yes, it is documented at onnxruntime/lora/adapter_format